app/tests/test_search.py

```diff
@@ -73,33 +72,87 @@ def test_search_results_displayed(client: FlaskClient):
     headers = header_row.find_all("th")
 
     expected_results_table = [
-        ["Transferring body", "Series", "Consignment reference", "File Name"],
+        ["Transferring body", "Series", "Consignment reference", "File name"],
         ["test body1", "test series1", "test consignment1", "test_file1.pdf"],
         ["test body2", "test series2", "test consignment2", "test_file2.txt"],
+        [
+            "testing body11",
+            "test series11",
+            "test consignment11",
+            "test_file11.txt",
+        ],
     ]
 
     assert [header.text for header in headers] == expected_results_table[0]
     for row_index, row in enumerate(results_rows):
         assert [
             result.text for result in row.find_all("td")
         ] == expected_results_table[row_index + 1]
+    # check pagination
+    assert b'aria-label="Page 1"' in response.data
 
 
-@patch("app.main.db.queries.db")
-def test_fuzzy_search_exception_raised(db, capsys):
+def test_search_results_displayed_multiple_pages(client: FlaskClient, app):
```

Comment:
we need the equivalent tests for browse

Reply:
there unit test cases created in browse for first , middle and last page - test_browse_display_first_page, test_browse_display_middle_page and test_browse_display_last_page

Reply:
ah that file diff didnt show in github by default due to it being a big amount of changes. I see it now